doc/zephyr.doxyfile.in

```diff
@@ -923,13 +926,16 @@ WARN_LOGFILE           =
 
 INPUT                  = @ZEPHYR_BASE@/doc/_doxygen/mainpage.md \
                          @ZEPHYR_BASE@/doc/_doxygen/groups.dox \
-                         @ZEPHYR_BASE@/kernel/include/kernel_arch_interface.h \
-                         @ZEPHYR_BASE@/include/zephyr/arch/cache.h \
-                         @ZEPHYR_BASE@/include/zephyr/sys/atomic.h \
                          @ZEPHYR_BASE@/include/ \
+                         @ZEPHYR_BASE@/kernel/include/kernel_arch_interface.h \
                          @ZEPHYR_BASE@/lib/libc/minimal/include/ \
                          @ZEPHYR_BASE@/subsys/testsuite/include/ \
-                         @ZEPHYR_BASE@/subsys/testsuite/ztest/include/
+                         @ZEPHYR_BASE@/subsys/testsuite/ztest/include/ \
```

Comment:
```suggestion
                         @ZEPHYR_BASE@/subsys/testsuite/ztest/include/
```

Reply:
I did this so that the last entry doesn't need to be updated when adding a new one at the bottom, keeping the Git blame history easier to look at.

Reply:
doesn't this mean Doxygen expects more stuff on the new line?

Reply:
+1, AFAICT these multiline entries need to be "terminated", but maybe they fixed it

Reply:
I'll try removing those trailing backslashes to see if that changes anything. I was thinking they would just be like trailing whitespaces because the next line is empty.

Reply:
Didn't help. :face_with_diagonal_mouth:
> Generating output.
Processing file doc/_doxygen/mainpage.md
Processing file doc/_doxygen/groups.dox
genhtml: ERROR: cannot read /home/runner/_work/zephyr/zephyr/include/kernel_arch_interface.h
Processing file include/kernel_arch_interface.h
Error: Process completed with exit code 2.

https://github.com/zephyrproject-rtos/zephyr/actions/runs/10417119532/job/28850735111